web-app/src/components/Game.js

```diff
@@ -62,6 +64,21 @@ export default class Game extends React.Component {
 		this.joinWithPreferences = this.joinWithPreferences.bind(this);
 		this.leave = this.leave.bind(this);
 		this.refinePhaseMessage = this.refinePhaseMessage.bind(this);
+
+		this.gamePromise = () => {
+			return helpers
+				.safeFetch(
+					helpers.createRequest(
+						"/Game/" + this.props.match.params.gameId
+					)
+				)
+			.then((resp) => resp.json());
+		}
+
+		this.close = () => {
```

Comment:
How come this is an anon func defined in the constructor and not a regular class method?

Reply:
I changed it.